Contributing.md

```diff
@@ -1,102 +1,104 @@
-# 💥 How to Contribute

```

Comment:
I'm wondering...
Can running prettier on the `Contributing.md` file have potential downsides?

Reply:
That doesn't run on any other file that public/data/*.JSON 🙂

Reply:
oh, the that's fine